include/boost/gil/image_processing/threshold.hpp

```diff
@@ -85,13 +85,13 @@ void threshold_binary
     {
         detail::threshold_impl<source_channel_t, result_channel_t>(src_view, dst_view,
             [threshold_value, max_value](source_channel_t px) ->
-                result_channel_t { return px >= threshold_value ? max_value : 0; });
+                result_channel_t { return px > threshold_value ? max_value : 0; });
```

Comment:
Would you mind reformatting all the lambdas to look this way?

```
[threshold_value, max_value](source_channel_t px) -> result_channel_t {
    return px > threshold_value ? max_value : 0;
});
```

This form is a bit more friendly for debugging, when one wants to set a breakpoint in the actual line
`return px > threshold_value ? max_value : 0;`.